docs/dev/build_android.md

```diff
@@ -5,54 +5,121 @@ This article describes how to build OpenVINO for Android operating systems.
-    -DANDROID_ABI=x86_64 \
-    -DANDROID_PLATFORM=21 \
-    -DANDROID_STL=c++_shared
+  unzip $OPV_HOME_DIR/platform-tools-latest-linux.zip -d $OPV_HOME_DIR
+  export ANDROID_TOOLS_PATH=$OPV_HOME_DIR/platform-tools
+  ```
 
-  make --jobs=$(nproc --all)
+_For Windows and Mac operating systems, download and unpack steps are similar_
+
+### Build and install OneTBB™
+To improve parallelism performance of the OpenVINO™ library using OneTBB, it is required to separately build OneTBB for a specific version of andoird-ndk:
+  ```bash
+  # Clone OneTBB™ repository 
+  git clone --recursive https://github.com/oneapi-src/oneTBB $OPV_HOME_DIR/one-tbb
+  # Create build and install directory 
+  mkdir $OPV_HOME_DIR/one-tbb-build $OPV_HOME_DIR/one-tbb-install
+  # Configure OneTBB™ CMake project 
+  cmake -S $OPV_HOME_DIR/one-tbb \
+        -B $OPV_HOME_DIR/one-tbb-build \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX=$OPV_HOME_DIR/one-tbb-install \
+        -DCMAKE_TOOLCHAIN_FILE=$ANDROID_NDK_PATH/build/cmake/android.toolchain.cmake \
+        -DANDROID_ABI=$CURRENT_ANDROID_ABI \
+        -DANDROID_PLATFORM=$CURRENT_ANDROID_PLATFORM \
+        -DANDROID_STL=$CURRENT_ANDROID_STL \
+        -DTBB_TEST=OFF \
+        -DCMAKE_SHARED_LINKER_FLAGS="-Wl,--undefined-version" 
+  # Build OneTBB™ project 
+  cmake --build $OPV_HOME_DIR/one-tbb-build --parallel
+  # Install OneTBB™ project 
+  cmake --install $OPV_HOME_DIR/one-tbb-build
   ```
 
-  * `ANDROID_ABI` specifies target architecture:
-    * `x86_64` for x64 build
-    * `armeabi-v7a with NEON` for ARM with NEON support
-    * `arm64-v8a` for ARM 64 bits
-  * `ANDROID_PLATFORM` - Android API version
-  * `ANDROID_STL` specifies that shared C++ runtime is used. Copy `~/Downloads/android-ndk/sources/cxx-stl/llvm-libc++/libs/x86_64/libc++_shared.so` from Android NDK along with built binaries
+### Build and install OpenVINO™
+More information about building OpenVINO™ for android systems can be found [here](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build_android.md).
+  ```bash
+  # Clone OpenVINO™ repository 
+  git clone --recursive https://github.com/openvinotoolkit/openvino $OPV_HOME_DIR/openvino
+  # Create build and install directory 
+  mkdir $OPV_HOME_DIR/openvino-build $OPV_HOME_DIR/openvino-install
+  # Configure OpenVINO™ CMake project 
+  cmake -S $OPV_HOME_DIR/openvino \
+        -B $OPV_HOME_DIR/openvino-build \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DCMAKE_INSTALL_PREFIX=$OPV_HOME_DIR/openvino-install \
+        -DENABLE_SAMPLES=ON \
+        -DENABLE_TESTS=ON \
+        -DCMAKE_TOOLCHAIN_FILE=$ANDROID_NDK_PATH/build/cmake/android.toolchain.cmake \
+        -DANDROID_ABI=$CURRENT_ANDROID_ABI \
+        -DANDROID_PLATFORM=$CURRENT_ANDROID_PLATFORM \
+        -DANDROID_STL=$CURRENT_ANDROID_STL \
+        -DTBB_DIR=$OPV_HOME_DIR/one-tbb-install \
+        -DTBBROOT=$OPV_HOME_DIR/one-tbb-install
```

Comment:
we can keep only `TBB_DIR` to point to folder with cmake files.

Reply:
@ilya-lavrenov corrected